src/main/java/org/mskcc/limsrest/service/sequencingqc/UpdateLimsSampleLevelSequencingQcTask.java

```diff
@@ -240,16 +240,15 @@ private JSONObject getStatsFromDb() {
     private String getIgoId(String id) {
         log.info("Stats IGO ID: " + id);
         List<String> idVals = Arrays.asList(id.split("_IGO_"));
-        if (id.contains(POOLEDNORMAL_IDENTIFIER) || id.contains(CONTROL_IDENTIFIER)){
-            if(idVals.size() == 2){
+        if (idVals.size() == 2) {
```

Comment:
could you comment why this is the case? 

Like, do the 2 ID values need to be POOLEDNORMAL_IDENTIFIER & CONTROL_IDENTIFIER?